src/core/mount-setup.c

```diff
@@ -390,9 +390,15 @@ int mount_setup(bool loaded_policy) {
                 nftw("/dev/shm", nftw_cb, 64, FTW_MOUNT|FTW_PHYS|FTW_ACTIONRETVAL);
                 nftw("/run", nftw_cb, 64, FTW_MOUNT|FTW_PHYS|FTW_ACTIONRETVAL);
 
+                /* Temporarily remount the root cgroup filesystem to give it a proper label. */
+                (void) mount("tmpfs", "/sys/fs/cgroup", "tmpfs", MS_REMOUNT|MS_NOSUID|MS_NOEXEC|MS_NODEV|MS_STRICTATIME, "mode=755");
```

Comment:
this shouldn't be done in fully unified mode. i.e. use cg_all_unified() to ensure we do this only if there's actually a tmpfs mounted there (in fully unified mode there's a cgroupfs directly mounted on /sys/fs/cgroup).

also, please specify the minimal parameters necessary. i.e. both "tmpfs" strings should be passed as NULL, as they do not (and cannot) change during a remount.